docker/app.base.dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM joomla:3.9-php7.1-apache as joomla_base
+
+LABEL maintainer="Yoti SDK <sdksupport@yoti.com>"
```

Comment:
Should we change this to websdk@yoti.com?

Reply:
We don't usually add this line so have removed it for now